playbooks/app_dev/binary_deployment_howto.adoc

```diff
@@ -0,0 +1,197 @@
+
+=== Creating the Build
+
+So for the build step, I require a few things:
+
+- My local WAR file
+- The name of the link:https://docs.openshift.com/enterprise/3.1/architecture/core_concepts/builds_and_image_streams.html#image-streams[ImageStream] for the image I want to deploy to. In this case, tomact 8.
+
+Since I already have my WAR file ready, I just need to find the right ImageStream to use. By default, OpenShift stores all of it's ImageStreams in a globally readable project called `openshift`, so I'll just do a search there for `tomcat8`.
+
+[source,bash]
+----
+[esauer@localhost warfile-demo]$ oc get imagestream -n openshift | grep tomcat8
+jboss-webserver30-tomcat8-openshift   registry.access.redhat.com/jboss-webserver-3/webserver30-tomcat8-openshift     1.1-3,1.1-7,1.2 + 2 more...        5 weeks ago
+----
+
+Now I can take that ImageStream name and plug it into a new build. I'm going to give the build a name of `ticket-monster`:
+
+[source,bash]
+----
+[esauer@localhost warfile-demo]$ oc new-build jboss-webserver30-tomcat8-openshift --name=ticket-monster --binary=true
+--> Found image 298446b (11 weeks old) in image stream "jboss-webserver30-tomcat8-openshift" in project "openshift" under tag "latest" for "jboss-webserver30-tomcat8-openshift"
+
+    JBoss Web Server 3.0
+    --------------------
+    Platform for building and running web applications on JBoss Web Server 3.0 - Tomcat v8
+
+    Tags: builder, java, tomcat8
+
+    * A source build using binary input will be created
+      * The resulting image will be pushed to image stream "ticket-monster:latest"
+
+--> Creating resources with label build=ticket-monster ...
+    imagestream "ticket-monster" created
+    buildconfig "ticket-monster" created
+--> Success
+----
+
+Notice what OpenShift did with my request... It created me an `imagestream` and a `buildconfig` object, both named `ticket-monster`. The `imagestream` will track the new images that get built as part of this new build process, and the `buildconfig` contains all the instructions that tell OpenShift how to build my app.
+
+Now I can kick off my build by pointing the `oc` client at my local project directory. The command to do that is `oc start-build <buildconfig-name> --from-file=<file-path>`. Before I run it though, I want to walk through a few parameters I'm going to use. First off, I know that my `<buildconfig-name>` will be `ticket-monster` and the `<file-path>` will be my current `warfile-demo` directory. If I run that command as-is (i.e. `oc start-build ticket-monster --from-file=.`) then a build will kick-off, the command will exit, and the build will run in the background. However, in this instance, I would prefer to actually follow the status of the build. To do that, I'm going to add the `--follow=true` (follow the logs of the build) and `--wait=true` (wait until the build completes to return an exit code).
```

Comment:
Did you want to use `--from-dir=` instead of `--from-file=` since you created the `deployments` directory?